Source/WebKit/ChangeLog

```diff
@@ -1,3 +1,48 @@
+2022-05-09  Kate Cheney  <katherine_cheney@apple.com>
+
+        Align ITP client side cookie cap with other script-writable storage
+        https://bugs.webkit.org/show_bug.cgi?id=240248
+        rdar://90468671
+
+        Reviewed by NOBODY (OOPS!).
+
+        * NetworkProcess/Classifier/ResourceLoadStatisticsDatabaseStore.cpp:
+        (WebKit::ResourceLoadStatisticsDatabaseStore::merge):
+        (WebKit::ResourceLoadStatisticsDatabaseStore::logCrossSiteLoadWithLinkDecoration):
+        (WebKit::ResourceLoadStatisticsDatabaseStore::setIsScheduledForAllScriptWritableStorageRemoval):
+        (WebKit::ResourceLoadStatisticsDatabaseStore::registrableDomainsToDeleteOrRestrictWebsiteDataFor):
+        (WebKit::ResourceLoadStatisticsDatabaseStore::setIsScheduledForAllButCookieDataRemoval): Deleted.
+        * NetworkProcess/Classifier/ResourceLoadStatisticsDatabaseStore.h:
+        * NetworkProcess/Classifier/ResourceLoadStatisticsStore.cpp:
+        (WebKit::domainsToString):
+        (WebKit::ResourceLoadStatisticsStore::didCreateNetworkProcess):
+        (WebKit::ResourceLoadStatisticsStore::setAgeCapForClientSideCookies): Deleted.
+        (WebKit::ResourceLoadStatisticsStore::updateClientSideCookiesAgeCap): Deleted.
+        * NetworkProcess/Classifier/ResourceLoadStatisticsStore.h:
+        * NetworkProcess/Classifier/WebResourceLoadStatisticsStore.h:
+        (WebKit::RegistrableDomainsToDeleteOrRestrictWebsiteDataFor::isolatedCopy const):
+        (WebKit::RegistrableDomainsToDeleteOrRestrictWebsiteDataFor::isolatedCopy):
+        (WebKit::RegistrableDomainsToDeleteOrRestrictWebsiteDataFor::isEmpty const):
+        Rename "non cookie website data" to "script writable storage" now that
+        we include client side cookies. This can't be done to the actual
+        database entry without a migration to a new database due to SQLite
+        restrictions around renames, so we will leave that one in place.
```

Comment:
The naming is OK and I believe there's already such naming in there. But with the new classification of cookies, I'd say script-written is more accurate than script-writeable. :)

Reply:
I can make the change for these new re-names! Better to be more accurate where we can IMO.